src/util-affinity.c

```diff
@@ -126,13 +126,11 @@ void BuildCpusetWithCallback(const char *name, ConfNode *node,
                 exit(EXIT_FAILURE);
             }
             if (b > max) {
-                SCLogError("%s: upper bound (%ld) of cpu set is too high, only %d cpu(s)", name, b,
+                SCLogError("%s: upper bound (%d) of cpu set is too high, only %d cpu(s)", name, b,
```

Comment:
`%u` now that `b` is `uint32_t`. Surprising no CI check rejected this.

Reply:
So, should I fix that in the next PR tackling other warnings ?

Reply:
yeah, staging branch with this PR is running